bin/reth/src/args/pruning_args.rs

```diff
@@ -40,6 +41,19 @@ impl PruningArgs {
                             .into_iter()
                             .collect(),
                     ),
+                    storage_history_filter: StorageHistoryPruneConfig(
+                        chain_spec
+                            .deposit_contract
+                            .as_ref()
+                            .map(|contract| {
+                                (
+                                    AddressAndSlots { address: contract.address, slots: vec![] },
+                                    PruneMode::Before(contract.block),
+                                )
+                            })
+                            .into_iter()
+                            .collect(),
+                    ),
```

Comment:
Yes it's correct what you assumed. It saves only the deposit contract storage from pruning. I actually think it's not needed , but I was not sure so I left it as you can see.

Let me know if I have to delete it

Reply:
i don't think it's doing anything, even if we had the need for it (we don't), since we have this:
`storage_history: Some(PruneMode::Distance(MINIMUM_PRUNING_DISTANCE)),`

Reply:
yes, you're right, thanks. I'll delete this and push the commit very soon